conditional/src/main/java/com/linecorp/conditional/ConditionMatchResult.java

```diff
@@ -152,13 +156,27 @@ public long startTimeMillis() {
         return startTimeMillis;
     }
 
+    /**
+     * Returns {@link OffsetDateTime} from the {@link ConditionMatchResult#startTimeMillis}.
+     */
+    public OffsetDateTime startTime() {
+        return Instant.ofEpochMilli(startTimeMillis).atZone(ZoneId.systemDefault()).toOffsetDateTime();
+    }
```

Comment:
Thanks for the great suggestion! In my opinion, this method is not necessary. Because the conversion from milliseconds to OffsetDateTime doesn't have to be done here. Because developers using Conditional can do it themselves. What do you think of using the method below??

```java
private static String millisAsISO8601String(long millis) {
    return OffsetDateTime.ofInstant(Instant.ofEpochMilli(millis), ZoneId.systemDefault()).toString();
}
```

Reply:
So, summarize your opinion...

1. `ConditionMatchResult.startTimeMillis`, `ConditionMatchResult.endTimeMillis` properties remain as `long` type as before.
2. `ConditionMatchResult.toString()` outputs `startTime` and `endTime` in `OffsetDateTime` format.

This seems like a good way. The values for `startTime` and `endTime` can be obtained through the public methods `startTimeMillis()` and `endTimeMillis()`, and We can know that the returned values are milliseconds through these method names, so users can return them as appropriate Date objects as needed 🙂 

However, I think it is something to consider about the mismatch between the type provided in the log(`OffsetDateTime`) and the type actually provided(`long`). 

Also, to use `OffsetDateTime`, the user must separately call the same method(you suggested) used to convert long to `OffsetDateTime` inside `ConditionMatchResult`.

So, if there's conversion from `long` to `OffsetDateTime` inside `ConditionMatchResult`, I think it's okay to add this value as a field and provide it for use as a public method.

What do you think?

Reply:
It seems we don't have to provide a public method to convert from Milliseconds -> OffsetDateTime. Because we need ZoneId or ZoneOffset to convert from Milliseconds to OffsetDateTime. Then, we need to receive ZoneId or ZoneOffset as parameter, which is no different from converting Milliseconds -> OffsetDateTime directly by the developer.

Reply:
Thank you for your kind explanation.
As you said, it seems more reasonable to provide the millisecond information as before, and let the user use this to get the necessary Date object.

I'll fix on it based on what you commented on 🙂

Reply:
I applied the way you suggested at a7c1734 🙂